demos/speech_recognition_streaming_demo/python/README.md

```diff
@@ -12,12 +12,17 @@ The application accepts
 * n-gram language model file in kenlm quantized binary format, and
 * an audio file in PCM WAV 16 kHz mono format.
 
-After computing audio features, running a neural network to get per-frame character probabilities, and CTC decoding, the demo prints the decoded text together with the timings of the processing stages.
+The application has two modes:
+
+ * *Offline mode* (default). Audio data is fed in 10 second chunks into a pipeline of: computing audio features, running a neural network to get per-frame character probabilities, and CTC decoding. After processing the whole file, the demo prints the decoded text and the time spent.
```

Comment:
for `streaming` demo having default `offline` mode looks strange

Reply:
To avoid confusion I renamed "offline" and "online" modes in demo to "normal" and "simulated real-time" modes.

Ex-"offline" mode is still streaming. "Streaming" means the all pipeline stages are executed for each new chunk of data, as opposed to the non-streaming pipeline processing (not employed in this demo), where first stage is executed for all data, then second stage for all data, etc. The difference between "(ex-)offline streaming" and "non-streaming" implementations is: 1) no need to accumulate input data in memory for the long files -- the processed input data can be dismissed, 2) smaller latency between providing the last chunk of data and getting the final result for the long files (e.g. few minutes audio length).

And the demo was renamed to "Speech Recognition DeepSpeech Demo".